enterprise/api/v1/views.py

```diff
@@ -308,21 +406,10 @@ def courses(self, request, enterprise_customer, pk=None):  # pylint: disable=inv
 
         Only courses with active course runs are returned. A course run is considered active if it is currently
         open for enrollment, or will open in the future.
-        ---
-        serializer: serializers.CourseSerializerExcludingClosedRuns
         """
         catalog_api = CourseCatalogApiClient(request.user)
         courses = catalog_api.get_paginated_catalog_courses(pk, request.GET)
-
-        # if API returned an empty response, that means pagination has ended.
-        # An empty response can also means that there was a problem fetching data from catalog API.
```

Comment:
See above for some reasoning, but for this comment in particular -- it kind of just repeats what's in the error message. Unless you mean to not pass in a customized error message and leave the comment, then I understand that the comment could be useful.